internal/setup/base.go

```diff
@@ -80,6 +80,7 @@ type BaseDendrite struct {
 const HTTPServerTimeout = time.Minute * 5
 const HTTPClientTimeout = time.Second * 30
 
+const NoInternalListener = ""
 const NoExternalListener = ""
```

Comment:
What's going on here? You need some actual values here else `NoInternalListener` is the same as `NoExternalListener` - https://play.golang.org/p/bIAL8JumJtK

Reply:
They're just designed to be human-readable rather than leaving `""` littered about the place, e.g. 
https://github.com/matrix-org/dendrite/blob/759bab6a0aa59c92122f330db25784eaad43967e/cmd/dendrite-monolith-server/main.go#L164-L168
https://github.com/matrix-org/dendrite/blob/759bab6a0aa59c92122f330db25784eaad43967e/cmd/dendrite-room-server/main.go#L35-L39

Reply:
Maybe `setup.NoListener` instead?